docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -191,6 +191,12 @@ To filter column values, click the filter (funnel) icon next to a column title.
 
 Click the check box next to the values that you want to display. Enter text in the search field at the top to show those values in the display so that you can select them rather than scroll to find them.
 
+Choose from several operators to display column values:
+
+- `contains` operator matches a regex pattern (operator by default)
+- The typical comparison operators (`=`, `!=`, `<`, `<=`, `>`, `>=`)
+- `expression` operator evaluates a bool expression, char v represents the column value in the expression (e.g. "v >= 10 && v <= 12").
```

Comment:
I know the order I've put the options in here isn't the order in which they show up in the UI, but I think this ordering will make for easier reading/comprehension.

```suggestion
- **Contains** - Matches a regex pattern (operator by default).
- **Expression** - Evaluates a boolean expression. The character `v` represents the column value in the expression (for example, "v >= 10 && v <= 12").
- The typical comparison operators: `=`, `!=`, `<`, `<=`, `>`, `>=`.
```